### PR TITLE
Gui - formatting + start of translations

### PR DESCRIPTION
**Pull request by woskov**

### CODE REVIEW

File: src/tunserver.cpp [hunk 1]
```diff
@@ -1080,7 +1080,7 @@ nlohmann::json c_tunserver::rpc_sending_test(const string &input_json) {
 nlohmann::json c_tunserver::rpc_add_peer(const string &input_json) {
 	auto input = nlohmann::json::parse(input_json);
 	auto peer = input["peer"].get<std::string>();
-	auto format = input["format"].get<std::string>();
+        auto format = input["format"].get<std::string>();
```
Comment (rfree-d) [inline src/tunserver.cpp]: bad ident

File: src-gui/quickstartdialog.h [hunk 2]
```diff
@@ -45,21 +45,21 @@ class quickStartDialog : public QWidget
...
+    void allowStranger(bool);			//!signal that allow strangers to net
+    void createNet();					//!signal create net
+    void connectNet(QString net);		//!signal connect net
+    void addUser(QString user_id);		//!signal that add user
+    void removeUser(QString user_id);	//!signal remove usera
 
 
 public slots:
-    void chooseInvate(QString invate);	//! slot dla sygnalu pobierajacego dane zaproszenia(np z ksiazki adresowej)
+    void chooseInvate(QString invate);	//! slot for option: chose invittaion from address book
```
Comment (rfree-d) [inline src-gui/quickstartdialog.h]: typo
"invitation"

File: src-gui/quickstartdialog.cpp [hunk 3]
```diff
@@ -37,42 +30,43 @@ quickStartDialog::~quickStartDialog()
...
-    if(ui->InviteEdit->text().isEmpty()) {
+    if( ui->InviteEdit->text().isEmpty() ) {
         emit createNet();
     } else {
         QString ip = ui->InviteEdit->text();
-        ip = ip.split('-').at(0);
+        ip = ip.split( '-' ).at( 0 );
 
-        emit connectNet(ip);		//!@todo dorobic sprawdzenie czy wprowadzony tekst jest poprawny
+        emit connectNet( ip );		//!@todo check if ip is propere
```
Comment (rfree-d) [inline src-gui/quickstartdialog.cpp]: typo "proper"

File: src-gui/meshpeer.h [hunk 4]
```diff
@@ -12,35 +12,33 @@ class MeshPeer : public QObject
     QString m_vip;
     QString m_net_name;
 
-
 public:
 
-
     enum COMMANDSTATUS {
```
Comment (robertoleksy) [inline src-gui/meshpeer.h]: enum class

File: src-gui/peersmodel.cpp [hunk 5]
```diff
@@ -5,262 +5,280 @@
...
+        if( data( idx ).toString() == value ) return idx;
     }
     return QModelIndex();
 }
+
+
+void peersModel::deletaAllPeers()
+{
+    beginRemoveRows( index( 0 ),0,m_peers_list.size()-1 );
+    foreach ( auto it, m_peers_list ) {
```
Comment (robertoleksy) [inline src-gui/peersmodel.cpp]: use c++11 range loop

File: src-gui/statusform.cpp [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+#include "ui_statusform.h"
+
+
+StatusForm::StatusForm( QWidget *parent ) :
+    QWidget( parent ),
+    ui( new Ui::StatusForm )
+{
+    ui->setupUi( this );
+    connect( &m_timer,SIGNAL( timeout() ),this,SLOT( onTickTimmer() ) );				// no check ping timmer yet
+    connect( &m_reconnectTimmer,SIGNAL( timeout() ),this,SLOT( onReConnectTimmer() ) );	//no reconnect timer yet
```
Comment (rfree-d) [inline src-gui/statusform.cpp]: typo

timmer -> timer
Timmer -> Timer

File: src-gui/nodecontrolerdialog.cpp [hunk 7]
```diff
@@ -3,87 +3,87 @@
...
-    m_executor = new commandExecutor(mw) ;//commandExecutor::construct(std::make_shared<MainWindow>(this));//std::make_shared<commandExecutor>(new commandExecutor(this));	//commandExecutor::construct(ret);
-    m_sender = new CommandSender(m_executor,mw);
-    m_executor->setSender(m_sender);
+    m_executor = new commandExecutor( mw ) ; //commandExecutor::construct(std::make_shared<MainWindow>(this));//std::make_shared<commandExecutor>(new commandExecutor(this));	//commandExecutor::construct(ret);
+    m_sender = new CommandSender( m_executor,mw );
+    m_executor->setSender( m_sender );
     QString host;
 
-    m_executor->startConnect(QHostAddress(address), port);		//! @todo zastanowic sie nad zrywaniem poprzedniego polaczenia
+    m_executor->startConnect( QHostAddress( address ), port );		//! @todo think abaout last send
```
Comment (rfree-d) [inline src-gui/nodecontrolerdialog.cpp]: typo 

about

**Comment by rfree-d** [comment]
@woskov could you remove the copied file "src-gui/trivialserialize.cpp" (and .hpp if there) as now the project can just use the trivialserialize.cpp / .hpp from main project?

**Comment by rfree-d** [comment]
Latest commit here is:

Commits on Oct 17, 2017

    fix in files after remove trivialserialize
    woskov committed 17 minutes ago

removing copy of trivialserialize.cpp
woskov committed 27 minutes ago

but github doesn't show it here (bug?)

**Pull request merged**